democracy_club/settings/base.py

```diff
@@ -242,8 +242,8 @@
 }
 
 GO_CARDLESS_PAYMENT_NAME = "Democracy Club Donation"
-GO_CARDLESS_PAYMENT_DESCRIPTION = "Helping Democracy Club increase the quantity,"\
-" quality and accessibility of information on election candidates, politicians and democratic processes"
+GO_CARDLESS_PAYMENT_DESCRIPTION = "Helping Democracy Club "\
+"increase the quality of information on elections & the democratic processes"
 GOCARDLESS_REDIRECT_URL = "https://democracyclub.org.uk/donate/thanks/"
```

Comment:
One more comment - when I tested locally, this unexpectedly dumped me back to the live site rather than my local dev site. I get that we have to pass through a fully qualified URL to gocardless, but if it is relatively easy to assemble  `GOCARDLESS_REDIRECT_URL = BASE_URL + "/donate/thanks/"` that would be nicer. If that is non-trivial, lets just ignore this point for now and get it deployed.